app/lending/components/LendingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import { Skeleton, Spinner } from '@heroui/react'
+import { useCallback, useEffect, useState } from 'react'
+import { useColumnsWithValues } from '../hooks/useColumnsWithValues'
+import { useLendingState } from '../LendingState'
+import { useScreenWidth } from '../hooks/useScreenWidth'
+import { AssetTable } from './AssetTable'
+import { MarketStats } from './market-stats/MarketStats'
+import { Header } from './Header'
+import { FormModal } from './form/FormModal'
+import { FormTab } from './form/types'
+import { MobileMarketList } from './mobile/MobileMarketList'
+import SkeletonPage from './SkeletonPage'
+
+export function LendingPage() {
+  const { skeleton } = useTestSkeleton(false)
+
+  const [loading, columns] = useColumnsWithValues()
+  const { setFormData, markets, fetching } = useLendingState()
+  const { screenLTE } = useScreenWidth()
+  const isMobileLayout = screenLTE('mobile')
```

Comment:
We already have hooks like `isMobileOrSmaller`. Why use this function from hook?